doc/usersguide/source/conf.py

```diff
@@ -31,12 +31,15 @@
 #extensions = ['sphinx.ext.todo', 'sphinx.ext.pngmath', 'rst2pdf.pdfbuilder']
 extensions = ['sphinx.ext.extlinks']
 
-numpydoc_baseurl = 'http://docs.scipy.org/doc/numpy/reference/generated'
+numpydoc_baseurl = 'https://numpy.org/doc/1.18/reference/'
 extlinks = {
     'issue': ('https://github.com/liam2/liam2/issues/%s', 'issue '),
     'pr': ('https://github.com/liam2/liam2/pull/%s', 'pull request '),
-    'numpy': (numpydoc_baseurl + '/numpy.%s.html', 'numpy.')
-}
+    # For base numpy (no occurence)
+    # 'numpy': (numpydoc_baseurl + 'generated/numpy.%s.html', 'numpy.'),
+    # Works only for numpy.random module
+    'numpy': (numpydoc_baseurl + 'random/generated/numpy.%s.html', 'numpy.'),
```

Comment:
Since this is specific to the random module, I would rather have a different "prefix": "numpy_random" or something like that, you could then also uncomment the general "numpy" prefix above.

Reply:
It was my first attempt then I realized that the only links using this feature where pointing to `numpy.random` ...

Reply:
Yes, I know but that could change, and having a "deceptive" link (numpy pointing to numpy random) could create problems down the line.